packages/validation/src/validators/admission-policy/validator.ts

```diff
@@ -321,4 +341,8 @@ export class AdmissionPolicyValidator extends AbstractPlugin {
       level,
     });
   }
+
+  private evaluateCelExpression(expression: string, params: string) {
```

Comment:
Let's leave it for now but making the Params a `Record<string, any>` and shifting down the encoding (yaml.stringify) would make this a tiny bit better abstraction.